ppml/trusted-big-data-ml/scala/docker-occlum/Dockerfile

```diff
@@ -41,7 +41,8 @@ RUN cd /opt && \
     rm spark/jars/spark-kubernetes_2.12-$SPARK_VERSION.jar && \
     rm spark/jars/spark-network-common_2.12-$SPARK_VERSION.jar && \
     rm spark/jars/hadoop-common-3.2.0.jar && \
-    rm spark/jars/hive-exec-2.3.7-core.jar
+    rm spark/jars/hive-exec-2.3.7-core.jar && \
```

Comment:
and /opt/spark/jars/log4j-1.2.17.jar?
and need replacements

Reply:
Updated.